titles.plugin.zsh

```diff
@@ -28,7 +28,7 @@ function _zsh_title__preexec() {
     fg)	cmd="${(z)jobtexts[${(Q)cmd[2]:-%+}]}" ;;
     %*)	cmd="${(z)jobtexts[${(Q)cmd[1]:-%+}]}" ;;
   esac
-  update_title "$cmd" "%20<...<%~"
+  update_title "$cmd" "%20<...<%~">/dev/null
```

Comment:
This actually ends up breaking this module in tmux.  Without `>/dev/null`, window titles correctly match the program running, but with this patch, tmux window titles no longer update appropriately.